discretize/mixins/mpl_mod.py

```diff
@@ -0,0 +1,2070 @@
+from __future__ import print_function
+import numpy as np
+import warnings
+from ..utils import mkvc, ndgrid, requires
+from ..utils.code_utils import deprecate_method
+from six import integer_types
+
+import matplotlib
+import matplotlib.pyplot as plt
+from matplotlib.widgets import Slider
+import matplotlib.colors as colors
+import matplotlib.cm as cmx
+from matplotlib.collections import PolyCollection, LineCollection
+from matplotlib import rc_params
+from mpl_toolkits.mplot3d.art3d import Line3DCollection
+import matplotlib.transforms as mtransforms
+import discretize
+
+class InterfaceMPL(object):
+    """This class is used for simple ``discretize`` mesh plotting using matplotlib.
+
+    This interface adds three methods to the meshes. ``plot_grid`` will plot the
+    grid points of each mesh in 2D and 3D. ``plot_image`` for 2D image ploting of
+    models. and ``plot_slice`` for plotting a 2D slice through a 3D mesh.
+    """
+
+    def plot_grid(
+        self, ax=None, nodes=False, faces=False, centers=False, edges=False,
+        lines=True, show_it=False, **kwargs
```

Comment:
I would vote to deprecate `show_it` - at least I fail to see the use-case (except for in the old-style gallery examples, but they are all going to change eventually, right?)

Reply:
(This applies to all functions, not just `plot_grid`.)